### PR TITLE
net-nds/389-ds-base: Revbump to add build patch provided by upstream

### PR DESCRIPTION
**Pull request by anomaly256**
Adds AC_PROG_CC_STDC to configure.ac

Closes bug 602718

Acked-by: wibrown@redhat.com

Package-Manager: portage-2.3.2

### CODE REVIEW

**Comment by anomaly256** [comment]
@gentoo/proxy-maint

File: net-nds/389-ds-base/389-ds-base-1.3.5.15-r1.ebuild [hunk 1]
```diff
@@ -51,6 +51,8 @@ pkg_setup() {
 }
 
 src_prepare() {
+	epatch "${FILESDIR}/389-ds-base-1.3.5-stdc.patch"
```
Comment (gktrk) [inline net-nds/389-ds-base/389-ds-base-1.3.5.15-r1.ebuild]: Can you attach the bug number here (e.g. `#602718`)

File: net-nds/389-ds-base/files/389-ds-base-1.3.5-stdc.patch [hunk 2]
```diff
@@ -0,0 +1,13 @@
+diff --git a/configure.ac b/configure.ac
```
Comment (gktrk) [inline net-nds/389-ds-base/files/389-ds-base-1.3.5-stdc.patch]: Can you attach the following to the header?

```
commit 60a4fa0e415bff73516c6d6ba44768d9719bf9f2
Author:     William Brown <firstyear@redhat.com>
AuthorDate: Thu Dec 15 09:48:11 2016 +1000
Commit:     William Brown <firstyear@redhat.com>
CommitDate: Thu Dec 15 11:09:04 2016 +1000

    Ticket 49068 - 1.3.5 use std c99
    
    Bug Description:  When we enabled the cve fix for constant time password comp,
    we included code that relied on a c99 feature. But we did not add the c99 flags
    to configure.ac. This has broken some distributions ports.
    
    Fix Description:  Add the AC_PROG_CC_STDC to configure.ac, and run autoreconf.
    
    https://fedorahosted.org/389/ticket/49068
    
    Author: wibrown
    
    Review by: mreynolds (thanks!)
```

**Comment by gktrk** [comment]
Does this resolve mjo's problem reported in https://bugs.gentoo.org/show_bug.cgi?id=602718#c6 ?

**Comment by anomaly256** [comment]
There was a new build error, William knows about it but has been swamped over xmas.  Will be tended to soon I'm sure

**Comment by Firstyear** [comment]
https://bugs.gentoo.org/show_bug.cgi?id=602718#c7 Details my response. I believe that the error they are encountering is a unique issue, and I am unable to reproduce it. I do not think we can block the inclusion of this PR much longer as it is a CVE fix, and a build fix, just because one person has a bad environment.

Thanks,

**Comment by gktrk** [comment]
Merged in https://github.com/gentoo/gentoo/compare/d2ef837...70895bc. Thanks!

**Comment by gktrk** [comment]
I missed it too but we should use the `Gentoo-Bug:` tag in commit messages as much as possible.

**Pull request closed without merging**